app/code/core/Mage/CatalogRule/Model/Action/Index/Refresh.php

```diff
@@ -118,7 +118,7 @@ public function execute()
 
         /** @var Mage_Core_Model_Date $coreDate */
         $coreDate  = $this->_factory->getModel('core/date');
-        $timestamp = $coreDate->gmtTimestamp('Today');
+        $timestamp = Mage::app()->getLocale()->date(null, null, null, true)->get(Zend_Date::TIMESTAMP);
```

Comment:
should this change be included in the PR?

Reply:
Oh sorry this was an accident on my side because I was doing something else. Didn't know this wasn't merged in so long ⭕

Let me reset the branch for 1.9.4.x